entrypoint.sh

```diff
@@ -88,14 +88,19 @@ function composer_install_dependencies {
         COMPOSER_ARGS="${COMPOSER_ARGS} --ignore-platform-req=php"
     fi
 
+
     case $DEPS in
         lowest)
             echo "Installing lowest supported dependencies via Composer"
+            # Disable platform.php, if set
+            composer config --unset platform.php
```

Comment:
Fancy, didn't know of this shortcut!

Reply:
@rdohms taught me to avoid manually updating `composer.json` whenever possible, and the `config` subcommand is our friend here. :)

Better, it doesn't emit an error if the setting isn't present, which ensures the script continues running in those cases.